ship-it-inc/ship-it-fend
# #167898467 Set up SCSS

#### What does this PR do?
It adds scss to the project
#### Description of Task to be completed?
- set up scss
- add variable colours
#### How should this be manually tested?
- Fetch the branch `ch-set-up-sass-167898467`
- Check out into the branch `ch-set-up-sass-167898467`
- run npm run dev
- The app should be started and the color of the displayed sentence should be the primary color
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
[#167898467](https://www.pivotaltracker.com/story/show/167898467)
#### Questions:
N/A
